.changeset/modern-cups-arrive.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-graph": patch
+"@finos/legend-studio": patch
+---
+
+Removed toGrammarString & fromGrammarString pair for service test runner
```

Comment:
Can we be more descriptive on why we are doing this. We should also link up the engine issues. Changesets will be part of the change logs when we do a release so think about what your change is doing overall. The message should explain why the change was made.